charters/DataStore/charter.md

```diff
@@ -0,0 +1,72 @@
+
+# Data Store
+
+
+## Description
+The Data Store is a scientific data sharing/publishing/distribution framework, providing file/bundle management on multiple clouds at petabyte-scale. It defines public APIs for storage, retrieval, and subscription to events that functions transparently across multiple cloud systems such as AWS and GCP.
+
+## Objective
+The objective of the Data Store group is to deliver substantively complete functionality on all of the in-scope items listed in this charter.
```

Comment:
Would this be generally true for all charters? If we approach objective this way it does not add information to the charter.

Reply:
Honestly, I wasn't sure what to put here. Now that there are other charters I will take a look to see if I can get inspired!